ui/static/ui/js/taxonomy/taxonomy_component.jsx

```diff
@@ -48,6 +48,16 @@ define('taxonomy_component', ['react', 'lodash', 'jquery',
           };
           vocabularies = vocabularies.concat([newVocab]);
         }
+        if (!_.isEmpty(vocabularies)) {
+          // sort vocabularies
+          vocabularies.sort(function (vocabLeft, vocabRight) {
+            if (vocabRight) {
```

Comment:
What does this check do? It seems like this should always be true, so we don't need this check here